shared/src/business/useCases/trialSessions/generatePrintableTrialSessionCopyReportInteractor.test.ts

```diff
@@ -0,0 +1,144 @@
+const {
+  applicationContext,
+} = require('../../test/createTestApplicationContext');
+const {
+  generatePrintableTrialSessionCopyReportInteractor,
+} = require('./generatePrintableTrialSessionCopyReportInteractor');
+const { ROLES } = require('../../entities/EntityConstants');
+
+describe('generatePrintableTrialSessionCopyReportInteractor', () => {
+  let mockUser;
+  let mockTrialSession;
+
+  const TRIAL_SESSION = {
```

Comment:
Nit and half an idea: could try using test data in `mockTrial` here instead - might need to add some fields though since it looks like the structure isn't quite the same

Reply:
I tried using `MOCK_TRIAL_REGULAR` from `mockTrial.js` and that seems to work

Reply:
Done